### PR TITLE
Feature/Remote Control - Radio and Climate Parameter Update

### PR DESCRIPTION
**Pull request by LuxoftAKutsan**
Fixes :  https://github.com/smartdevicelink/sdl_core/issues/2793 

PR is **ready** for review

This proposal changes the minimum index of HD radio sub-channels from 1 to 0.
In addition, we propose to add a new parameter climateEnable to ClimateControlData which will allow an application to power climate control on or off.

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: docs/Common/Structs/index.md [hunk 1]
```diff
@@ -504,14 +504,15 @@
 |frequencyFraction|Integer|false|minvalue:0 maxvalue:9|The fractional part of the frequency for 101.7 is 7|
 |band|Common.RadioBand|false|||
 |rdsData|Common.RdsData|false|||
-|availableHDs|Integer|false|minvalue:1 maxvalue:7|number of HD sub-channels if available|
-|hdChannel|Integer|false|minvalue:1 maxvalue:7|Current HD sub-channel if available|
+|availableHDs|Integer|false|minvalue:0 maxvalue:7|number of HD sub-channels if available <br> deprecated="true" since="5.1"|
```
Comment (yang1070) [inline docs/Common/Structs/index.md]: need update since="5.1" to a proper number

File: docs/Common/Structs/index.md [hunk 2]
```diff
@@ -504,14 +504,15 @@
...
+|hdChannel|Integer|false|minvalue:0 maxvalue:7|Current HD sub-channel if available. since="5.1|
```
Comment (yang1070) [inline docs/Common/Structs/index.md]: 5.1 --> 5.2?

File: docs/Common/Structs/index.md [hunk 3]
```diff
@@ -504,14 +504,15 @@
...
 |signalStrength|Integer|false|minvalue:0 maxvalue:100||
 |signalChangeThreshold|Integer|false|minvalue:0 maxvalue:100|If the signal strength falls below the set value for this parameter, the radio will tune to an alternative frequency|
 |radioEnable|Boolean|false||True if the radio is on, false is the radio is off|
 |state|Common.RadioState|false||||
 |sisData|Common.SisData|false||Read-only Station Information Service (SIS) data provides basic information about the station such as call sign, as well as information not displayable to the consumer such as the station identification number|
 |hdRadioEnable|Boolean|false||True if the hd radio is on, false is the radio is off|
+|availableHdChannels|Integer|false|minvalue:0 maxvalue:7 array="true" minsize="0" since="5.1"|the list of available hd sub-channel indexes, empty list means no Hd channel is available, read-only|
```
Comment (yang1070) [inline docs/Common/Structs/index.md]: 5.2?

**Comment by KhrystynaDubovyk** [comment]
@yang1070, please see the updates https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/153/commits/0a24c0dc4b589cfc3bdc95bbfc134e64af3b4562

**Pull request merged**